verilog/tools/ls/vscode/src/download-ls.ts

```diff
@@ -105,14 +123,13 @@ export async function checkAndDownloadBinaries(
       file.path = path.basename(file.path);
       return file;
     },
-    plugins: platform() === "win32" ? [] : [decompressTargz()],
+    plugins: platform() === "win32" ? [decompressUnzip()] : [decompressTargz()],
```

Comment:
we implicitly assume that the binary is zipped for windows and tar.gz'ed for the rest. Might be useful to use the uncompression method based on the file-ending (fine for now, just might be something for the future).

Reply:
I'll leave this one for now.